example/flutter/pubspec.yaml

```diff
@@ -0,0 +1,6 @@
+name: usage_example
+dependencies:
+  usage:
+    path: ../..
+  flutter:
+    path: ../../../flutter/packages/flutter
```

Comment:
cc @abarth for an example of why we need a way to connect a package from the ecosystem to flutter :)